binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/pom.xml

```diff
@@ -84,6 +84,43 @@
 			<artifactId>awaitility</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>context-propagation</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-tracing-bridge-brave</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-tracing-integration-test</artifactId>
```

Comment:
We probably can just look into `io.micrometer:micrometer-observation-test` and that one for Brave instead of this huge one and all those exclusions.